src/doc/DocAnnotator.js

```diff
@@ -788,12 +788,32 @@ class DocAnnotator extends Annotator {
      * @return {void}
      */
     drawingSelectionHandler(event) {
-        if (this.modeControllers[TYPES.draw]) {
-            this.modeControllers[TYPES.draw].handleSelection(event);
+        const controller = this.modeControllers[TYPES.draw];
+        if (controller && !this.isCreatingAnnotation() && !this.isCreatingHighlight) {
```

Comment:
Do we eventually want to move to an event model where we broadcast the beginning and end of creating an annotation? Then we don't have to worry about checking modes and highlights we can just look at global annotation state based on said events.

Reply:
Yes